uniffi/Cargo.toml

```diff
@@ -14,7 +14,7 @@ keywords = ["ffi", "bindgen"]
 # Re-exported dependencies used in generated Rust scaffolding files.
 anyhow = "1"
 bytes = "1.0"
-ffi-support = "~0.4.2"
+ffi-support = "~0.4.3"
```

Comment:
This gets us an automatic `impl IntoFfi for c_void`, ref https://github.com/mozilla/ffi-support/pull/4